skimage/feature/tests/test_peak.py

```diff
@@ -445,3 +445,15 @@ def test_input_labels_unmodified(self):
                                      min_distance=1, threshold_rel=0,
                                      indices=False, exclude_border=False)
         assert np.all(labels == labelsin)
+
+    def test_many_objects(self):
+        mask = np.zeros([500, 500], dtype=bool)
+        x, y = np.indices((500, 500))
+        x_c = x // 20 * 20 + 10
+        y_c = y // 20 * 20 + 10
+        mask[(x - x_c) ** 2 + (y - y_c) ** 2 < 8 ** 2] = True
+        labels, num_objs = ndi.label(mask)
+        dist = ndi.distance_transform_edt(mask)
+        local_max = peak.peak_local_max(dist, min_distance=20, indices=True,
+                                        exclude_border=False, labels=labels)
+        assert (len(local_max) == 625)
```

Comment:
```suggestion
        assert len(local_max) == 625
```

just triggering a build

Reply:
Maybe just apply this to trigger a fresh new build. I think builds are fixed.